packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  getFocusStyle,
+  HighContrastSelector
+} from '../../Styling';
+import {
+  ILinkStyleProps,
+  ILinkStyles
+} from './Link.types';
+
+export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
+  const { disabled, theme } = props;
+  const { palette, semanticColors } = theme;
```

Comment:
shouldn't it warn you for unused "palette" here?

Reply:
It didn't, but I'm glad you did.